apps/coscrad-frontend/src/store/slices/resources/books/thunks/fetch-books.ts

```diff
@@ -0,0 +1,10 @@
+import { createFetchThunk } from '../../../utils/createFetchThunk';
+import { RESOURCES } from '../../constants';
+import { getApiResourcesBaseRoute } from '../../shared';
+import { BOOK } from '../constants';
+import { BookIndexState } from '../types';
+
+export const fetchBooks = createFetchThunk<BookIndexState>(
+    `${RESOURCES}/${BOOK}/fetch`,
+    `${getApiResourcesBaseRoute()}/books`
```

Comment:
We could either
- have a helper to manage API routes on the frontend
- put this in the api-interfaces layer (not sure if this is best practice and whether it is unwanted coupling)